csuntechlab/iSTART
# STR-193 -- FE: Implement multi-module support

# Description
Implemented multiple module support
  
# Testing
1. Visit the dashboard page.
2. Ensure when clicking Marijuana, it leads to a test module.
